### ISSUE TITLE
[bug/help] Need to import `mapbox.css` to see map

### ISSUE THREAD

**Issue by Maker-Mark** [geospoc/v-mapbox#378]
I'm having issues getting the map to show up at all unless I manually import the v-mapbox css like so: `import "v-mapbox/dist/v-mapbox.css";`
I don't see this as part of the getting started guide. Is there something I'm missing?

When I do import the CSS this way, I get a parcel (v1) warning about the source map
`⚠️  Could not load existing sourcemap of "../node_modules/v-mapbox/dist/v-mapbox.css".`

**Comment by karol-bujacek** [comment]
@Maker-Mark, the `v-mapbox.css` file is mentioned in the code snipped in this section: https://v-mapbox.netlify.app/guide/#installation-2

You are right that it might be listed also in the previous snippet – it is easy to overlook it.

**Comment by Maker-Mark** [comment]
@karol-bujacek Yes, I'm not using the package using the CDN, I'm using it in ES Modules.
should it be:
```
import 'v-mapbox/dist/v-mapbox.css'
import 'mapbox-gl/dist/mapbox-gl.css'
import Mapbox from 'mapbox-gl'
import { MglMap } from 'v-mapbox'
```
?

I'm not sure since when I use it directly, I see:
When I do import the CSS this way, I get a parcel (v1) warning about the source map
⚠️ Could not load existing sourcemap of "../node_modules/v-mapbox/dist/v-mapbox.css".

**Comment by karol-bujacek** [comment]
@Maker-Mark, you’ve motivated me to check the code and take a deeper look into a console in my browser (Firefox).

I am not seeing the message you’ve mentioned, but several other:

* This page appears to be missing CSS declarations for Mapbox GL JS, which may cause the map to display incorrectly. Please ensure your page includes mapbox-gl.css, as described in https://www.mapbox.com/mapbox-gl-js/api/. mapbox-gl.js:30:4608

* Source map error: Error: NetworkError when attempting to fetch resource.
Resource URL: webpack:///./node_modules/mapbox-gl/dist/mapbox-gl.js?
Source Map URL: mapbox-gl.js.map

* Source map error: Error: NetworkError when attempting to fetch resource.
Resource URL: webpack:///./node_modules/mapbox-gl/dist/mapbox-gl-csp-worker.js?
Source Map URL: mapbox-gl-csp-worker.js.map

I suspect that the first one is caused by wrong order of loading the JS and CSS files. Other two are probably (?) caused by wrong webpack configuration.

Nevertheless, the application works correctly. Do you encounter some problems related to the quoted message?

**Comment by adambergman** [comment]
+1 from me. I didn't get an error in my console on Chrome but I wasn't able to see the map until I imported the style sheet from v-mapbox.

I noticed if I did not import the styles from mapbox-gl, I would see a map but things weren't styled. When I imported the mapbox-gl stylesheet the map would disappear. After quite some time I finally found this issue and it pointed me to import the v-mapbox styles and things worked perfectly.

Maps visible but unstyled/not functional
```javascript
import { MglMap, MglMarker, MglPopup, MglAttributionControl } from 'v-mapbox';
```

Component completely invisible, but wrapper in DOM
```
import 'mapbox-gl/dist/mapbox-gl.css';
import { MglMap, MglMarker, MglPopup, MglAttributionControl } from 'v-mapbox';
```

Works as expected
```
import 'mapbox-gl/dist/mapbox-gl.css';
import 'v-mapbox/dist/v-mapbox.css';
import { MglMap, MglMarker, MglPopup, MglAttributionControl } from 'v-mapbox';
```

This is with Vue 2.6.12 and Webpack 4.
